libs/ml/include/ml/core/graph.hpp

```diff
@@ -18,20 +18,9 @@
 //------------------------------------------------------------------------------
 
 #include "ml/core/node.hpp"
-#include "ml/meta/ml_type_traits.hpp"
 #include "ml/ops/weights.hpp"
```

Comment:
I think weights are not used anywhere in the header - so can be deleted or moved to source

Reply:
If I remove "ml/ops/weights" then I need include these headers:

 #include "ml/regularisers/regulariser.hpp"
#include "ml/exceptions/exceptions.hpp"